### ISSUE TITLE
lower global timeout again (210 -> 150), raise ceiling for 2.12 only

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-jenkins-infra#101]
see https://github.com/scala/scala-jenkins-infra/pull/100 for details

**Comment by adriaanm** [comment]
I've been thinking about this, and I like the idea of setting no timeout by default, and having job-specific timeouts for jobs where we expect them to sometimes not end.

**Comment by adriaanm** [comment]
I'll submit a PR to concretize my thoughts in a couple of minutes.

**Comment by adriaanm** [comment]
looking at https://scala-ci.typesafe.com/job/scala-2.11.x-validate-test/buildTimeTrend, since we've limited concurrency to 2, no build took more than 1h47min

**Comment by adriaanm** [comment]
Hm, the concurrency change is more recent, so I should've said, in the last 2 months, no build took 2hrs.

**Comment by adriaanm** [comment]
we should also really configure job concurrency at a category level:
- cpu-heavy (test/ide/community build): 2 per node

hudson.plugins.throttleconcurrents.ThrottleJobProperty.xml:

```
<?xml version='1.0' encoding='UTF-8'?>
<hudson.plugins.throttleconcurrents.ThrottleJobProperty_-DescriptorImpl plugin="throttle-concurrents@1.8.4">
  <categories>
    <hudson.plugins.throttleconcurrents.ThrottleJobProperty_-ThrottleCategory>
      <maxConcurrentPerNode>2</maxConcurrentPerNode>
      <maxConcurrentTotal>0</maxConcurrentTotal>
      <categoryName>cpu-heavy</categoryName>
      <nodeLabeledPairs/>
    </hudson.plugins.throttleconcurrents.ThrottleJobProperty_-ThrottleCategory>
  </categories>
</hudson.plugins.throttleconcurrents.ThrottleJobProperty_-DescriptorImpl
```

**Comment by SethTisue** [comment]
this is n/a anymore after #103 was merged and #102 was opened
